appveyor.yml

```diff
@@ -23,20 +23,34 @@ environment:
       EXAMPLE: 01-executable
       CONFIG: Release
 
+    - TOOLCHAIN: "vs-14-2015"
```

Comment:
We are testing them already, can be removed.

Reply:
Comment or remove?

Reply:
Removed.